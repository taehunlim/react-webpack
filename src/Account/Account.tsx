import React from 'react';

const Account = ({name, mbti}: {name: string, mbti: string}) => {
    return (
        <div>
            <h2>{name}</h2>
            <p>{mbti}</p>
        </div>
    );
};

export default Account;