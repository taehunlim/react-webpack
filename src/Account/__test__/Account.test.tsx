import React from 'react';
import Account from "../Account";
import {render} from "@testing-library/react";

it("matches snapshot", () => {
    const utils = render(<Account name="호박 고구마" mbti="ENTP"/>);
    expect(utils.container).toMatchSnapshot();
})

it("show the props correctly", () => {
    const utils = render(<Account name="호박 고구마" mbti="ENTP"/>);
    utils.getByText("호박 고구마");
    utils.getByText("ENTP")
})