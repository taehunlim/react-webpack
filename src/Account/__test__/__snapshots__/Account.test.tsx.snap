// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <h2>
      호박 고구마
    </h2>
    <p>
      ENTP
    </p>
  </div>
</div>
`;
